metron-platform/metron-management/src/test/java/org/apache/metron/management/EnrichmentConfigFunctionsTest.java

```diff
@@ -92,15 +84,15 @@ public void setup() {
     }
     return ret;
   }
-  private int size(Map<String, Object> stellarFunctions) {
+  private int size(Map<String, Object> stellarFunctions, String group) {
```

Comment:
It's because it was `@Parameterized` before. They were getting set in the constructor as fields, but with the change in semantics, they aren't now, so it got adjusted.